tests/vmi_configuration_test.go

```diff
@@ -3391,3 +3391,28 @@ func createBlockDataVolume(virtClient kubecli.KubevirtClient) (*cdiv1.DataVolume
 
 	return virtClient.CdiClient().CdiV1beta1().DataVolumes(testsuite.GetTestNamespace(nil)).Create(context.Background(), dataVolume, metav1.CreateOptions{})
 }
+
+func GetProcessName(pod *k8sv1.Pod, pid string) (string, error) {
```

Comment:
Following the function re-location, it could not be unexported / private to the package (start with a lower case).

Reply:
fixed